djcelery_email/tasks.py

```diff
@@ -17,6 +17,15 @@
 TASK_CONFIG = {'name': 'djcelery_email_send_multiple', 'ignore_result': True}
 TASK_CONFIG.update(settings.CELERY_EMAIL_TASK_CONFIG)
 
+# import base if string to allow a base celery task
+if 'base' in TASK_CONFIG and isinstance(TASK_CONFIG['base'], str):
```

Comment:
Correct me if I'm wrong, but I believe it's best to use `basestring` as the type for an `isinstance` check.